index.html

```diff
@@ -337,13 +338,14 @@ <h1>Get Started</h1>
                     <div class="email">
                         <label for="email">E-mail</label>
                         <br>               
-                        <input type="text" id="email" name="email" placeholder="Your e-mail..">
+                        <input type="text" id="email" name="_replyto" placeholder="Your e-mail..">
                     </div>
                 </div>
                 <label for="textarea">Message</label>
-                <textarea name="textarea" id="textarea" cols="30" rows="10">Your message here ...</textarea>
+                <textarea name="message" id="textarea" cols="30" rows="10" placeholder="Your message here ..."></textarea>
```

Comment:
- [ ] The `text area` should have the `max length` attribute to activate the validation on the form submit. Kindly fix it.

Reply:
Changed. Thanks for this review.